workingtitle-project-cj4.xml

```diff
@@ -7,5 +7,6 @@
 		<Package>PackageDefinitions\workingtitle-vcockpits-instruments-airliners.xml</Package>
 		<Package>PackageDefinitions\working-title-vcockpits-instruments.xml</Package>
 		<Package>PackageDefinitions\workingtitle-aircraft-cj4.xml</Package>
+        <Package>PackageDefinitions\workingtitle-vcockpits-instruments-navsystems.xml</Package>
```

Comment:
we really have to check for compatibility here. for what is it needed?

Reply:
This is used for the warning system. This is where the core warning system classes live, I just duplicated them and customized the code. It's just living in the same directory but has no collisions. All this is saying is include the custom file in that directory I made.